sourcegraph/sourcegraph.proto

```diff
@@ -588,6 +588,214 @@ service Repos {
 	rpc ListCommitters(ReposListCommittersOp) returns (CommitterList);
 }
 
+// StorageError represents an error when interacting with the Storage service.
+message StorageError {
+	// Code represents the type of error for programatic handling.
+	enum Code {
+		// None is the error code used when programatic handling of the error is not
+		// advised. It can be assumed that the operation failed, and that a human
+		// readable message is also provided.
+		None = 0;
+
+		// EOF is the error returned by Read when no more input is available.
+		// Functions should return EOF only to signal a graceful end of input. If
+		// the EOF occurs unexpectedly in a structured data stream, the appropriate
+		// error is either UNEXPECTED_EOF or some other error message giving more
+		// detail.
+		EOF = 1;
+
+		// NotExist is the error used when attempting to read/write to an object
+		// that does not exist.
+		NotExist = 2;
+
+		// Permission is the error used when permission to access the given file is
+		// not granted to you.
+		Permission = 3;
+	}
+
+	// code is the error code. If no error code is specified then programatic
+	// handling of the error is not advised. The user should be informed of the
+	// error message instead.
+	Code code = 1;
```

Comment:
For consistency, this should be `Code is ...`.

The pattern is "{{.VariableNameUsingGoStyle}} ...".